commit 0f40f4a21a10bf802860b27dba6bd48ad673deb7
Author: Akshu Agrawal <akshu.agrawal@amd.com>
Date:   Tue Aug 21 12:21:57 2018 +0530

    clk: x86: Set default parent to 48Mhz
    
    [ Upstream commit bded6c03e398dc6e862dc8301fb9a60175740653 ]
    
    System clk provided in ST soc can be set to:
    48Mhz, non-spread
    25Mhz, spread
    To get accurate rate, we need it to set it at non-spread
    option which is 48Mhz.
    
    Signed-off-by: Akshu Agrawal <akshu.agrawal@amd.com>
    Reviewed-by: Daniel Kurtz <djkurtz@chromium.org>
    Fixes: 421bf6a1f061 ("clk: x86: Add ST oscout platform clock")
    Signed-off-by: Stephen Boyd <sboyd@kernel.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/clk/x86/clk-st.c b/drivers/clk/x86/clk-st.c
index fb62f3938008..3a0996f2d556 100644
--- a/drivers/clk/x86/clk-st.c
+++ b/drivers/clk/x86/clk-st.c
@@ -46,7 +46,7 @@ static int st_clk_probe(struct platform_device *pdev)
 		clk_oscout1_parents, ARRAY_SIZE(clk_oscout1_parents),
 		0, st_data->base + CLKDRVSTR2, OSCOUT1CLK25MHZ, 3, 0, NULL);
 
-	clk_set_parent(hws[ST_CLK_MUX]->clk, hws[ST_CLK_25M]->clk);
+	clk_set_parent(hws[ST_CLK_MUX]->clk, hws[ST_CLK_48M]->clk);
 
 	hws[ST_CLK_GATE] = clk_hw_register_gate(NULL, "oscout1", "oscout1_mux",
 		0, st_data->base + MISCCLKCNTL1, OSCCLKENB,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1311_0f40f4a21a10bf802860b27dba6bd48ad673deb7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1311_0f40f4a21a10bf802860b27dba6bd48ad673deb7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
